assets/js/validation.js

```diff
@@ -1,21 +1,41 @@
 const axios = require('axios');
 
 let name = document.getElementById('name');
-let validationResult = document.getElementById('validation-result');
+let validationResultName = document.getElementById('validation-result-name');
+let team = document.getElementById('team');
+let validationResultTeam = document.getElementById('validation-result-team');
+
 const validateName = function () {
-    validationResult.innerText = '...';
-    axios.post(validationResult.dataset.path, {input: name.value})
-        .then(function(response) {
+    validationResultName.innerText = '...';
+    axios.post(validationResultName.dataset.path, {input: name.value})
+        .then(function (response) {
+            if (response.data.valid) {
+                validationResultName.innerHTML = "<img src=https://reactiongifs.me/wp-content/uploads/2019/02/Macho-Man-Smile.gif width=\"200px\" height=\"150px\" alt=\"alt\">";
+            } else {
+                validationResultName.innerHTML = "<img src=https://media1.giphy.com/media/nstGUYA6WnWJa/giphy.gif?cid=790b7611e958e0e51adbfce26cccda017a20761d1ac59c6d&rid=giphy.gif width=\"200px\" height=\"150px\" alt=\"alt\">";
+            }
+        })
+        .catch(function (error) {
+            validationResultName.innerText = 'Error: ' + error;
+        });
+};
+
+const validateTeam = function () {
+    validationResultTeam.innerText = '...';
+    axios.post(validationResultTeam.dataset.path, {input: team.value})
+        .then(function (response) {
             if (response.data.valid) {
-                validationResult.innerHTML = ":)";
+                validationResultTeam.innerHTML = "<img src=https://media.giphy.com/media/cJMmZY7jSCtwN4z0WX/giphy.gif width=\"200px\" height=\"150px\" alt=\"alt\">";
```

Comment:
:+1: už humoro jausmą
:-1: už papildomą darbą tikrinant, nes negaliu automatizuoti tikrinimo

Reply:
Be humoro – rankiniu būdu testai prasisuka

![image](https://user-images.githubusercontent.com/1453957/70867505-d77f3580-1f7e-11ea-859a-b73637e05a9d.png)